leapp/libraries/stdlib/__init__.py

```diff
@@ -3,9 +3,19 @@
 represents a location for functions that otherwise would be defined multiple times across leapp actors
 and at the same time, they are really useful for other actors.
 """
+import os
+import sys
+
 import six
 import subprocess
-import os
+import uuid
+
+from leapp.utils.audit import create_audit_entry
+from leapp.libraries.stdlib import call
+
+
+class CalledProcessError(Exception):
```

Comment:
This should be based on [leapp.exceptions.LeappError](https://leapp.readthedocs.io/en/latest/pydoc/leapp.html#leapp.exceptions.LeappError) and also contain stderr, stdout, return code etc

Reply:
@vinzenz  `LeappError` or `CommandError`? (the second one is used)

Reply:
@pirat89 LeappError is the baseclass of all leapp Exceptions